cpp-client/tests/test_util.cc

```diff
@@ -114,7 +115,11 @@ TableMakerForTests TableMakerForTests::create() {
 }
 
 Client TableMakerForTests::createClient() {
-  std::string connectionString("localhost:10000");
+  const char* hostptr = std::getenv("DH_HOST");
```

Comment:
Are we OK with "licking the cookie" on a short name like `DH_HOST` rather than say `DEEPHAVEN_HOST`?
I'm not particularly opinionated, just wondering if this was considered. I suppose if it's only for our own tests it doesn't matter much.

Reply:
The pre-existing via-docker test code (eg gradle files and supporting code for python, go, etc) use these variables already.
I think it makes sense to be consistent, and I don't think this PR would be the place to rename everything, so my humble take here is that we should take this conversation to a different PR if we think that value for effort that change would make sense.  For reference in making that decision:
```
cfs@guanaco 23:18:28 ~/dh/oss1/deephaven-core
$ find . -type f | xargs grep DH_HOST
./py/client/pydeephaven/session.py:            self.host = os.environ.get("DH_HOST", "localhost")
./py/client/build.gradle:        environmentVariable 'DH_HOST', deephavenDocker.containerName.get()
./py/client/build/buildWheel-docker/src/build.gradle:        environmentVariable 'DH_HOST', deephavenDocker.containerName.get()
./py/client/build/buildWheel-docker/src/pydeephaven/session.py:            self.host = os.environ.get("DH_HOST", "localhost")
./sphinx/build/cppClientDoxygen-docker/build.gradle:                  'DH_HOST=' + deephavenDocker.containerName.get() + ' ' +
./sphinx/build/cppClientDoxygen-docker/tests/test_util.cc:  const char* hostptr = std::getenv("DH_HOST");
grep: ./.gradle/7.5/executionHistory/executionHistory.bin: binary file matches
./cpp-client/build/testCppClient-docker/Dockerfile:ENV DH_HOST=dh-server-for-cpp-933abaa3-6584-4bfa-86f0-993add1082be
./cpp-client/build/testCppClient-docker/tests/test_util.cc:  const char* hostptr = std::getenv("DH_HOST");
./cpp-client/build/testCppClient-dockerfile/Dockerfile:ENV DH_HOST=dh-server-for-cpp-933abaa3-6584-4bfa-86f0-993add1082be
./cpp-client/tests/#test_util.cc#:  const char *hostptr = std::getenv("DH_HOST");
./cpp-client/tests/test_util.cc:  const char* hostptr = std::getenv("DH_HOST");
./cpp-client/build.gradle:        environmentVariable 'DH_HOST', deephavenDocker.containerName.get()
./go/build.gradle:        environmentVariable 'DH_HOST', deephavenDocker.containerName.get()
./go/README.md:Tests assume the server is on `localhost:10000` by default, but can be configured by setting the `DH_HOST` and `DH_PORT` environment variables.
./go/README.md:but can be configured by setting the `DH_HOST` and `DH_PORT` environment variables.
./go/internal/test_tools/test_tools.go:// By default it is localhost, but can be overriden by setting the DH_HOST environment variable.
./go/internal/test_tools/test_tools.go: host := os.Getenv("DH_HOST")
```

Reply:
It's fine, I was just bringing it up. You folks are the the deciders.